tests/drivers/pwm/pwm_api/boards/nrf54h20dk_nrf54h20_cpuapp.overlay

```diff
@@ -19,3 +19,7 @@
 	pinctrl-names = "default", "sleep";
 	memory-regions = <&cpuapp_dma_region>;
 };
+
+&pwm120 {
+	status = "disabled";
```

Comment:
I actually don't see why `pwm120` would be enabled in this test, but anyway, I don't think this is the proper solution for such problem. If you take a look at how this test selects the PWM instance for testing: https://github.com/zephyrproject-rtos/zephyr/blob/e32c2ee2968599e893172a0221bab7d4bde3632b/tests/drivers/pwm/pwm_api/src/test_pwm.c#L32-L42 a more appropriate thing seems to be to add:
```
/ {
	aliases {
		pwm-0 = &pwm130;
	};
};
```